src/Matrix.jl

```diff
@@ -3558,6 +3558,8 @@ If `side` is `:right` and `flag` is true, $B$ is the right inverse of $A$
 i.e. $AB$ is the $n\times n$ unit matrix. If `side` is `:left` and `flag` is
 true, $B$ is the left inverse of $A$ i.e. $BA$ is the $m\times m$ unit matrix.
 If `flag` is false, no right or left inverse exists.
+
+To get the space of all inverses, combine with [`nullspace`](@ref).
 """
 function is_invertible_with_inverse(A::MatrixElem{T}; side::Symbol = :left) where {T <: RingElement}
```

Comment:
Not an issue of this PR, but for the record: I am surprised that this function can do strictly more (namely handle non-square matrices) than its `is_invertible` sibling.